src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -198,13 +198,17 @@ private void renameSelected() {
             if (result.getNumber() == NicknamePokemonResponse.Result.SUCCESS_VALUE) {
                 success.increment();
                 System.out.println("Renaming " + PokeHandler.getLocalPokeName(pokemon) + " from \"" + pokemon.getNickname() + "\" to \"" + PokeHandler.generatePokemonNickname(renamePattern, pokemon) + "\", Result: Success!");
+            } else if (result.getNumber() == NicknamePokemonResponse.Result.UNSET_VALUE) { // FIXME: See PokeHandler.java@97
```

Comment:
One addition here. If you unset the nickname, it would hit here too and tell it was skipped, even if it was renamed.
I would add a ` && !pokemon.getNickname().equals(PokeHandler.generatePokemonNickname(renamePattern, pokemon))`. To check that the nickname it's the same.